R/createmodelutils.R

```diff
@@ -49,7 +49,7 @@ doFunction <- function(f, param_list) {
 #' @param model_obj model object to output
 #' @return key value pair list with name and model
 #' @export
-modelOut <- function(model_name, model_obj) {
+prepModelForOutput <- function(model_name, model_obj) {
```

Comment:
I love the name you came up with. We should use the verb `prep` to indicate such operations. Awesome :+1: